src/Microsoft.Azure.SignalR.Common/ClientInvocation/ClientInvocationManager.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ClientInvocationManager
```

Comment:
this looks only applicable for AspNetCore SignalR and let's move to SignalR project instead of in Common

Reply:
fixed